test/numerical/CMakeLists.txt

```diff
@@ -3,15 +3,15 @@
 add_custom_target(numerical)
 set_target_properties(numerical PROPERTIES FOLDER "Tests")
 
-add_custom_target(check-numerical
+add_custom_target(check-onnx-numerical
   COMMENT "Running the ONNX MLIR numerical regression tests"
-  COMMAND "${CMAKE_CTEST_COMMAND}" -L numerical --output-on-failure -C $<CONFIG> --force-new-ctest-process
+  COMMAND "${CMAKE_CTEST_COMMAND}" -j $$\{NPROC:-$$\(nproc\)\} -L numerical --output-on-failure -C $<CONFIG> --force-new-ctest-process
```

Comment:
This change needs to be reflected in the Windows CI as well - since it does not set NPROC, the tests are no longer running in parallel there.

Reply:
So why it didn't take the $(nproc) fallback? $(nproc) does not work on Windows? In that case, perhaps a more Windows compatible change is to make the fallback to be `${CTEST_PARALLEL_LEVEL}`.

Reply:
I am not sure what $(nproc) fallback you are referring to. The windows ci does not set an nproc variable for the numerical tests anywhere. The change that you made parallelizes the tests based on a variable that ends up having no set value in that case.

Reply:
`${NPROC:-$(nproc)}` means that if `NPROC` is not set, use `$(nproc)`.

Reply:
I thought you were saying that there was a fallback to `$(nproc)`, this makes more sense. There's no `$(nproc)` on Windows.

Reply:
> Would you like me to send a change for PR that would use a cmake variable for this?

No need. I modified the dockerfiles to set CTEST_PARALLEL_LEVEL for the numerical tests so no change required on Windows.